location-service/src/main/java/com/movie/recommender/location/model/dto/CountryWithCitiesDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.movie.recommender.location.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CountryWithCitiesDTO {
```

Comment:
move to `com.movie.recommender.common.model.location` so it can be reused by `crawler`